flashinfer/mla.py

```
@@ -453,3 +520,285 @@ def run(
+            query,
+            kv_cache,
+            qk_nope_head_dim,
+            kv_lora_rank,
+            qk_rope_head_dim,
+            sparse_mla_top_k,
+            block_tables,
+            block_size,
+        )
+
+        if out is None:
+            out_shape = query.shape[:-1] + (kv_lora_rank,)
```

Ruff (0.14.7)

Consider `(*query.shape[:-1], kv_lora_rank)` instead of concatenation

Replace with `(*query.shape[:-1], kv_lora_rank)`

(RUF005)